Gordon360/Services/NewsService.cs

```diff
@@ -34,17 +43,22 @@ public StudentNews Get(int newsID)
             {
                 throw new ResourceNotFoundException() { ExceptionMessage = "The news item was not found." };
             }
+
+            string imagePath = newsItem.Image;
+
```

Comment:
Since you're just returning the newsItem as is from the database, this is no longer necessary.